src/codeu/chat/common/RawController.java

```diff
@@ -36,6 +36,8 @@
   // use, the call will fail and null will be returned.
   User newUser(Uuid id, String name, Time creationTime);
 
+  User newNickname(Uuid id, String nickname, Time creationTime);
```

Comment:
i need the two methods, one method calls the other one that is how it is supposed to be :P